docs/router.md

```diff
@@ -0,0 +1,204 @@
+# About Routers
+
+`gpii.express.router` is a subclass of [`gpii.express.middleware`](middleware.md) that wraps
+[the  Express `router` object](http://expressjs.com/en/4x/api.html#router), which is simply a container for a collection
```

Comment:
Review the new docs in `handler.md` for a few ideas on alternate approaches and a fuller description of why we do things this way.  Hopefully that will give you an idea of how possible it is to move to another model.